src/web/mjs/engine/Enums.mjs

```diff
@@ -12,4 +12,8 @@ export default class Enums {
             onSelectManga:            'onSelectManga'
         };
     }
+
+    static get NovelColorProfiles() {
```

Comment:
Maybe more fitting to place this into `Settings.mjs`, since there are no defined constants at all and the options itself are already in settings

Reply:
yea i was following https://github.com/manga-download/hakuneko/pull/2975#discussion_r565007963 to literally